timed/employment/serializers.py

```diff
@@ -285,7 +282,7 @@ class Meta:
         """Meta information for the location serializer."""
 
         model = models.Location
-        fields = ["name", "workdays"]
+        fields = ("name", "workdays")
```

Comment:
we could hint all of these oneliners to be mulltilined by `ruff format`
```suggestion
        fields = ("name", "workdays",)
```

This would reduce future churn by simplifying diffs at the cost of making a already huge file huger

This comment is on the smallest edge case of 2 fields, on other serializers further down the trade off is clearer

Reply:
i've added trailing commas to all tuples i came across now

Reply:
neat! i think this will help alleviate the "f", "o", "o" issue with single value tuple even further while introducing consistency